tests/nightly/test_np_large_array.py

```diff
@@ -163,28 +149,27 @@ def test_all():
 
 @use_np
 def test_amin():
-    A = np.ones((INT_OVERFLOW, 2))
-    A[100][1] = -1
-    A.attach_grad()
+    inp = np.ones((INT_OVERFLOW, 2))
+    inp[-1, -1] = -1
+    inp.attach_grad()
```

Comment:
We may also later add the check about `req="add"`. But this can be done in a separate PR.

Reply:
@sxjscience didn't quite understand this. Can you explain it a bit more

Reply:
@access2rohit I've provided an example script in https://github.com/apache/incubator-mxnet/issues/19045 about how to correctly test an operator

Reply:
Basically, the test here is still not enough to ensure that it will work in the DL system. We will need to test for 1) `req=add`, 2) backward other than all ones.